lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -9,6 +9,9 @@ ShopifyApp.configure do |config|
   config.shop_session_repository = 'Shop'
   config.log_level = :info
   config.reauth_on_access_scope_changes = true
+  config.webhooks = [
+    { topic: "app/uninstalled", address: "webhooks/app_uninstalled"}
+  ]
```

Comment:
Notice I don't have any GDPR webhooks because they are just templates and don't actually do anything. It would be up to the developer to actually flush these jobs out.